webapp/channels/src/components/start_trial_form_modal/index.tsx

```diff
@@ -209,7 +218,7 @@ function StartTrialFormModal(props: Props): JSX.Element | null {
         }
         return {
             value: orgSize,
-            label: OrgSize[orgSize as unknown as keyof typeof OrgSize],
+            label: formatMessage({id: orgSize, defaultMessage: OrgSize[orgSize as unknown as keyof typeof OrgSize]}),
```

Comment:
Does this allow i18n-extract to pull the correct values here?

Very handy if that's the case. I ran into a situation where I needed to do this but figured it wasn't possible using variables since i18n-extract wouldn't update i18n files for me.

Reply:
Nope - it's taking the orgSize (which is a value equal to a key in the OrgSize enum) and fetching that enum key's value.

To fix what you ran into, I have [these](https://github.com/mattermost/mattermost-server/pull/22870/files#diff-a2bbf0442211f02ba4f58758282fd2b365ef56e975f1f74fdcceefad08314390R46-R51) - tells i18n-extract that they're used so it won't strip them out